tests/Unit/Core/Language/Pack/Loader/RemoteLanguagePackLoaderTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Unit\Core\Language\Pack\Loader;
+
+use PHPUnit\Framework\TestCase;
+use PrestaShop\PrestaShop\Core\Foundation\Version;
+use PrestaShop\PrestaShop\Core\Language\Pack\Loader\RemoteLanguagePackLoader;
+
+final class RemoteLanguagePackLoaderTest extends TestCase
```

Comment:
```suggestion
class RemoteLanguagePackLoaderTest extends TestCase
```

Reply:
:thinking: Why do you think it is better to not let this class `final` ??

Reply:
One or the other don't matter to me, personally, since there are very few chances that this class will be extended anyway

Reply:
And if it is, I don't think it would be a BC break to remove the final :thinking: